src/standalone/Vegetation/soil_drivers.jl

```diff
@@ -0,0 +1,35 @@
+using ..ClimaLSM.Soil
+export AbstractSoilDriver, PrescribedSoil, PrognosticSoil
+
+"""
+     An abstract type of soil drivers of the canopy model.
+"""
+abstract type AbstractSoilDriver{FT <: AbstractFloat} end
+
+"""
+     PrescribedSoil{} <: AbstractSoilDriver
+
+A container for holding prescribed soil parameters needed by the canopy model
+when running the canopy in standalone mode, including the soil pressure and 
+albedo.
+$(DocStringExtensions.FIELDS)
+"""
+struct PrescribedSoil{FT} <: AbstractSoilDriver{FT}
```

Comment:
What do you think about defining an outer constructor for this which takes these as keyword arguments? it might be a little clearer in usage.

Reply:
(note - the default inner constructor will still work, so you dont need to change the tests. maybe just the tutorials and experiments can use the kwargs constructor?)